index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome Section</a>
+      <a href="github.com/DiegoMero/Wilfried-Diego-Microverse" id="profile-link">My portfolio</a>
+    </nav>
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+    </section>
+
+    <section id="projects">
+      <div class="project-title"></div>
+    </section>
+
+
+
```

Comment:
Please consider not leaving unnecessary spaces between lines of code to have a more readable code.:+1::+1: